account_statement_import_file/tests/test_account_statement_import_file.py

```diff
@@ -38,7 +36,9 @@ def setUpClass(cls):
                 "bank_account_id": cls.bank_account.id,
             }
         )
-        f_path = file_path("account_statement_import_file/tests/test_file.txt")
+        f_path = file_path(
+            "account_statement_import_file/test_files/test_statement_import.txt"
+        )
```

Comment:
hi @mmequignon,
you're right, it should be in the `/tests/samples` folder because there is a convention for that type of file in https://github.com/OCA/bank-statement-import/blob/a919f9bab7187b8f89d66074c10535a61a0f4977/.pre-commit-config.yaml#L20
so on this occasion, i move it to the `/tests/samples` folder